crates/release_plz_core/src/release.rs

```diff
@@ -131,9 +137,26 @@ fn release_package(
         info!("published {}", package.name);
     }
 
+    if input.gh_release {
+        publish_release(git_tag(&package.name, &package.version.to_string()));
+    }
+
     Ok(())
 }
 
+async fn publish_release(git_tag: String) {
+    let owner = RepoUrl::new("https://www.github.com").unwrap().owner;
+    let repo = RepoUrl::new("https://www.github.com").unwrap().name;
+    let page = octocrab::Octocrab::default()
```

Comment:
we need to construct the GitHub client by using [this](https://github.com/MarcoIeni/release-plz/blob/36616bbb470e429c4686c41234f9a6b4b7dee936/crates/release_plz_core/src/github_client.rs#L15) method.
In this way, we can pass the personal access token.
Without the token, we can't create the release.
You need to read the github personal access token in input, similarly to how we do for the release-pr command [here](https://github.com/MarcoIeni/release-plz/blob/36616bbb470e429c4686c41234f9a6b4b7dee936/crates/release_plz/src/args/release_pr.rs#L17).